### ISSUE TITLE
A11y_SQLAzureDataStudio_NewDeployment_ SQLServerBigDataCluster_Screenreader: NVDA doesn’t announce as ‘Required’ for ‘I accept…License Terms’ checkbox

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9323]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
#A11y_SQLAzureDataStudio; #A11yMAS; #A11ySev2; #A11yTCS; #SQL Azure Data Studio; #Benchmark; #DesktopApp; #Win10; #MAS1.3.1 #Screenreader; #NVDA; #JAWS;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Additional Details:
MAS References: MAS1.3.1
NVDA Version: 2019.2

**Repro Steps:**
1.	Launch Azure Data Studio.
2.	Select the Connections icon (if not already selected) from left bar.
3.	Click on the ellipses '...' and select "New Deployment".
4.	Select "SQL Server Big Data Cluster" card under deployment options
5.	Launch NVDA screenreader
6.	Navigate to ‘I accept…License Terms’ checkbox using tab key

**Actual:**
NVDA doesn’t announce as ‘Required’ for ‘I accept…License Terms’ checkbox when navigated using tab key.

**Same issue is observed with JAWS screenreader**
**Same issue is observed in Database Dashboard screen:** 1. Launch Azure Data Studio.2. Connect to server.3. In the Connections pane, navigate to server --> database --> right click on any database & select Manage.

**Expected:**
NVDA should announce as ‘Required’ for ‘I accept…License Terms’ checkbox when navigated using tab key.

**User Impact:**
Screen reader users will not get information that ‘I accept…License Terms’ checkbox is mandatory field.

**Attachment link for Reference:**
[9323_A11y_SQLAzureDataStudio_NewDeployment_ SQLServerBigDataCluster_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274816/9323_A11y_SQLAzureDataStudio_NewDeployment_.SQLServerBigDataCluster_Screenreader.zip)

**Comment by mspatil110** [comment]
#A11ySev2

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
